### PR TITLE
🚢 Add launch blogpost

### PR DESCRIPTION
**Pull request by agoose77**
This PR adds the first JB blogpost!

### CODE REVIEW

File: posts/2024-11-01-jupyter-book-2-alpha.md [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+
+The MyST engine is on-track to support the inclusion of references and other markup features generated by code cells.
+
+### Control Cell Visibility with Tags
+
+In the beta release, Jupyter Book 2 will once-again be able to show and hide content according to cell tags.
+
+## Jupyter Book 2 vs MyST-MD
+
+% TODO testimonial from JB 1->2 users
```
Comment (choldgraf) [inline posts/2024-11-01-jupyter-book-2-alpha.md:52]: Don't worry about this testimonial stuff, too much overhead for an alpha blog post

File: posts/2024-11-01-jupyter-book-2-alpha.md [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+      - Jupyter Book
+abbreviations:
+  CLI: command-line interface
+license: CC-BY-4.0
+---
+
+Over the last ten months, the Jupyter Book team have been hard at work; Jupyter Book has [become a Jupyter subproject](https://github.com/jupyter/governance/pull/229), and the team[^jb-eb] announced a [plan to release Jupyter Book 2][plan]. This post announces the **alpha release of Jupyter Book 2.0**, which has been re-written from the ground up to use the new MyST-MD engine.
+
+Over the next few months, we will work in preparation for the full release of Jupyter Book 2. Stay tuned for more!
+
```
Comment (choldgraf) [inline posts/2024-11-01-jupyter-book-2-alpha.md:18]: Add a brief "instructions for trying out Jupyter Book 2.0 alpha"

File: posts/2024-11-01-jupyter-book-2-alpha.md [hunk 3]
```diff
@@ -0,0 +1,58 @@
+---
+title: Announcing Jupyter Book 2 alpha
```
Comment (choldgraf) [inline posts/2024-11-01-jupyter-book-2-alpha.md:2]: Remove any language like `2.0.0a0`. That is engineer language, user language is `Jupyter Book 2 alpha`

**Review by choldgraf — APPROVED** [review]
A few quick thoughts and I also pushed some changes via a commit. I think we should just get this out and not try to make it perfect.

File: posts/2024-11-01-jupyter-book-2-alpha.md [hunk 4]
```diff
@@ -0,0 +1,58 @@
...
+### Simple Instant Search
+A new client-side search uses a simple, modern algorithm for fast _local_ search that finds the results that you care about.
+
+% TODO: embed GIF / video
+
+### High Quality PDFs
+
+PDF documents can now be built with Typst, a high-quality typesetting engine that produces readable error messages and beautiful documents.
+
+## Coming Soon in `2.0.0b0`
```
Comment (choldgraf) [inline posts/2024-11-01-jupyter-book-2-alpha.md:36]: 2.0 beta

File: posts/2024-11-01-jupyter-book-2-alpha.md (hunk 2 above)
Comment (choldgraf) [reply posts/2024-11-01-jupyter-book-2-alpha.md:18]: Also add a call to action for where people can give feedback

File: posts/2024-11-01-jupyter-book-2-alpha.md (hunk 3 above)
Comment (agoose77) [reply posts/2024-11-01-jupyter-book-2-alpha.md:2]:
```suggestion
title: Announcing the Jupyter Book 2 alpha
```

File: posts/2024-11-01-jupyter-book-2-alpha.md [hunk 5]
```diff
@@ -0,0 +1,58 @@
...
+% TODO: embed example
+
+### Simple Instant Search
+A new client-side search uses a simple, modern algorithm for fast _local_ search that finds the results that you care about.
+
+% TODO: embed GIF / video
+
+### High Quality PDFs
+
+PDF documents can now be built with Typst, a high-quality typesetting engine that produces readable error messages and beautiful documents.
```
Comment (rowanc1) [inline posts/2024-11-01-jupyter-book-2-alpha.md:34]:
```suggestion
PDF documents can now be built with Typst, a high-quality typesetting engine that produces readable error messages and beautiful documents. This feature was the basis for the 2024 [SciPy proceedings](https://proceedings.scipy.org), which is now built on MyST Markdown and will be accepting Jupyter Notebooks in 2025.
```

image

File: posts/2024-11-01-jupyter-book-2-alpha.md [hunk 6]
```diff
@@ -0,0 +1,58 @@
...
+    affiliations:
+      - Jupyter Book
+abbreviations:
+  CLI: command-line interface
+license: CC-BY-4.0
+---
+
+Over the last ten months, the Jupyter Book team have been hard at work; Jupyter Book has [become a Jupyter subproject](https://github.com/jupyter/governance/pull/229), and the team[^jb-eb] announced a [plan to release Jupyter Book 2][plan]. This post announces the **alpha release of Jupyter Book 2.0**, which has been re-written from the ground up to use the new MyST-MD engine.
+
+Over the next few months, we will work in preparation for the full release of Jupyter Book 2. Stay tuned for more!
```
Comment (rowanc1) [inline posts/2024-11-01-jupyter-book-2-alpha.md:17]:
```suggestion
Over the next few months, we will work in preparation for the full release of Jupyter Book 2. Stay tuned for more! The initial documentation for the alpha release can be found at: \
<https://next.jupyterbook.org/>
```

File: posts/2024-11-01-jupyter-book-2-alpha.md (hunk 5 above)
Comment (rowanc1) [reply posts/2024-11-01-jupyter-book-2-alpha.md:34]: https://proceedings.scipy.org/articles/WXRH8633

**Review by rowanc1 — APPROVED** [review]
Looks great for an alpha announcement - lets get this out shortly after the release!

I think the only major thing is adding the how to give feedback.

**Pull request merged**